ssl/ech/ech_local.h

```diff
@@ -217,6 +268,33 @@ int ossl_ech_conn_init(SSL_CONNECTION *s, SSL_CTX *ctx,
 void ossl_ech_conn_clear(OSSL_ECH_CONN *ec);
 void ossl_echext_free(OSSL_ECHEXT *e);
 OSSL_ECHEXT *ossl_echext_dup(const OSSL_ECHEXT *src);
+#  ifdef OSSL_ECH_SUPERVERBOSE
+void ech_pbuf(const char *msg, const unsigned char *buf, const size_t blen);
+void ech_ptranscript(const char *msg, SSL_CONNECTION *s);
+#  endif
+int ech_get_retry_configs(SSL_CONNECTION *s, unsigned char **rcfgs,
+                          size_t *rcfgslen);
+int ech_send_grease(SSL_CONNECTION *s, WPACKET *pkt);
+int ech_pick_matching_cfg(SSL_CONNECTION *s, OSSL_ECHSTORE_ENTRY **ee,
+                          OSSL_HPKE_SUITE *suite);
+int ech_encode_inner(SSL_CONNECTION *s);
+int ech_find_confirm(SSL_CONNECTION *s, int hrr, unsigned char *acbuf,
+                     const unsigned char *shbuf, const size_t shlen);
+int ech_make_transcript_buffer(SSL_CONNECTION *s, int for_hrr,
+                               const unsigned char *shbuf, size_t shlen,
+                               unsigned char **tbuf, size_t *tlen,
+                               size_t *chend, size_t *fixedshbuf_len);
+int ech_reset_hs_buffer(SSL_CONNECTION *s, const unsigned char *buf,
+                        size_t blen);
+int ech_aad_and_encrypt(SSL_CONNECTION *s, WPACKET *pkt);
+int ech_swaperoo(SSL_CONNECTION *s);
+int ech_calc_confirm(SSL_CONNECTION *s, int for_hrr, unsigned char *acbuf,
+                     const unsigned char *shbuf, const size_t shlen);
+
+/* these are internal but located in ssl/statem/extensions.c */
+int ech_same_ext(SSL_CONNECTION *s, WPACKET *pkt);
+int ech_same_key_share(void);
+int ech_2bcompressed(int ind);
```

Comment:
All these functions need to have `ossl_` prefix

Reply:
think that's done now